src/ec.rs

```diff
@@ -2,100 +2,134 @@ pub use self::secp256k1::*;
 
 #[cfg(feature = "secp256k1-c")]
 mod secp256k1 {
-    /// Alias type for `secp256k1::Error`
-    pub type Error = secp256k1::Error;
+    /// `secp256k1::Error`
+    pub use secp256k1::Error;
+
+    lazy_static::lazy_static! {
```

Comment:
Do you mind if we remove it for this PR? I'd like to see some benchmarks that justify `lazy_static` first.

Reply:
no problem